asn1ate/sema.py

```diff
@@ -719,6 +721,19 @@ def __str__(self):
     __repr__ = __str__
 
 
+class Imports(SemaNode):
+    def __init__(self, elements):
+	self.module2symbols = { }
+	for i in range(0, len(elements), 2):
+		self.module2symbols [elements [i+1].elements[0]] = set(elements [i])
```

Comment:
I can't really get my head around this, and I think you underestimate the form of imports -- X.680 lists a couple of forms that I think this will fail to pick up;

    IMPORTS x FROM M;
    IMPORTS a, b, c FROM OtherModule;
    IMPORTS informationFramework
        FROM UsefulDefinitions {joint-iso-itu-t(2) ds(5) module(1) usefulDefinitions(0) 3};

etc.